user_guide_src/source/installation/upgrade_460.rst

```diff
@@ -85,6 +85,25 @@ Also, methods that returns an ``int`` still lose the microseconds.
 .. literalinclude:: upgrade_460/005.php
    :lines: 2-
 
+.. _upgrade-460-time-set-timestamp:
+
+Time::setTimestamp() Behavior Fix
+=================================
+
+In previous versions, if you call ``Time::setTimestamp()`` on a Time instance with
```

Comment:
Isn't it better to use this?
```suggestion
In previous versions, if you call :php:meth:`Time::setTimestamp()` on a Time instance with
```

Reply:
`:php:meth:` makes a reference to the class reference.
But there is no class reference for `Time::setTimestamp()` method.

*Class reference* is something like the following:
https://codeigniter4.github.io/CodeIgniter4/outgoing/view_renderer.html#namespace-CodeIgniter\View